platform/ios/test/MGLLayerTests.m

```diff
@@ -0,0 +1,25 @@
+#import "MGLMapViewTests.h"
```

Comment:
That’s fine; the custom style layer API is getting folded into the normal style layer API in #7250, so soon there won’t be anything extra to test.